tests/fixtures.py

```diff
@@ -1,14 +1,38 @@
 from datetime import datetime
+from pytz import UTC
```

Comment:
`pytz` is a third party library - can we put it in the same section as `faculty` and `mlflow`?

Reply:
👍